NewSpring/ApollosProject
# Style Updates

## DESCRIPTION

### What does this PR do, or why is it needed?
This PR updates some of the styles on the main prayer menu page to make it look and work better on more devices.

### How do I test this PR?
Run the prayer app in your sim and see if the changes work. Essentially, I updated the prayer menu page to use a `SafeAreaView` and messed around with some of the margins to make it work better, particularly on smaller phones. It's still going to scroll on them, but this looks better.

---

> I am affirming this is my _best_ work ([Ecclesiastes 9:10](https://www.bible.com/bible/97/ECC.9.10.MSG))

## TODO

- [n/a] Closes [DEV-XXX](#url)

## REVIEW

**Manual QA**


**Code Review: Questions to consider**


---

> The purpose of PR Review is to _improve the quality of the software._
